### PR TITLE
Add a doc to deploy hybrid CF across Azure and Azure Stack

### PR DESCRIPTION
**Pull request by bingosummer**

### CODE REVIEW

File: docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+1. Confiure DNS server to Azure DNS
+
+    1. Get your Name Server Domain Name. You can get four `nsdname` from the output.
+
+        ```
+        az network dns record-set ns show --resource-group ${PRIMARY_RESOURCE_GROUP_NAME} --zone-name ${ZONE_NAME} --name "@"
+        ```
+
+    1. Configure DNS server. It may differ from each provider, but the process is basically same. You can usually get the guidance in your domain-name service provider about how to configure your name server domain name.
```
Comment (andyliuliming) [inline docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md]: 2. ?

Comment (andyliuliming) [reply docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md]: and also for other places.

Comment (bingosummer) [reply docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md]: Numbered lists do not need to sequentially increase. For example, 
The following content
```
1. a
1. b
1. c
```
will be rendered to:
1. a
1. b
1. c

Comment (andyliuliming) [reply docs/advanced/hybrid-cloud-foundry-across-azure-and-azure-stack/README.md]: cool. learned that.

**Pull request merged**